src/components/common/Item.js

```diff
@@ -85,7 +85,7 @@ const Item = ({ id, isChildren }) => {
           <Typography className={FOLDER_NAME_TITLE_CLASS} variant="h2">
             {item.get('name')}
           </Typography>
-          {children.map((thisItem) => (
+          {children.filter(i => pinnedOnly && i.settings.isPinned).map((thisItem) => (
```

Comment:
Perhaps we want some security check with `i.settings?.isPinned`

Reply:
It doesn't change anything with or without `?.`, it will return a  `falsy` value if isPinned doesn't exists which means it isn't pinned.
And settings will always be at least an empty object.